gpos/xml/scap/ssg/content/ssg-chainguard-xccdf/OvalChecks/XccdfOvalChecks.xml

```diff
@@ -18,7 +19,7 @@
     <dc:identifier>Chainguard-GPOS-OVAL</dc:identifier>
     <dc:language>en-US</dc:language>
   </metadata>
-  <platform idref="cpe:/o:chainguard:chainguard_linux"/>
+  <model system="urn:xccdf:scoring:default"/>
```

Comment:
... and also, I don't think we should drop the chainguard specific cpe... though searching NIST's CPE database, I don't see a specific chainguard or wolfi cpe filed, so I guess we should ask for one.